vignettes/polars.Rmd

```diff
@@ -306,41 +306,44 @@ multiple variables and aggregation types.
 ```{r}
 dat$group_by(
   "cyl",
-  pl$col("am")$cast(pl$Boolean)$alias("manual")
+  manual = pl$col("am")$cast(pl$Boolean)
```

Comment:
I noticed that the column renaming by named Expr is not working here. It needs to be fixed.